### ISSUE TITLE
error when setting standard notifications

### ISSUE THREAD

**Issue by JChipault** [USGS-WiM/whispersservices_django#369]
Report from external user is below. This happened to him consistently yesterday and again today, but I haven't heard it reported from anyone else (though number of testers is pretty small right now):
Attached a screen capture of an error message I receive under Notifications > Manage Notification Preferences >  Standard Notifications and select any of the boxes to receive notifications 'new' or 'updated'. The error message blinks and never goes away until I refresh the browser window (which is google chrome).

Adding New Custom Notification Trigger was pretty slick and bug-free.

![image](https://user-images.githubusercontent.com/49287076/78711678-fe21fd00-78dc-11ea-80b6-7ffe260830c9.png)

**Comment by BlakeDraper** [comment]
@aaronstephenson This appears to be a backend problem - the PATCH to update the user's `new_notification_cue_standard_preferences` is failing for a role error. Admin and SuperAdmins seem fine, this is discovered from a PartnerManager role, but could affect others as well. 
![Capture](https://user-images.githubusercontent.com/6978380/78828521-71de0b80-79aa-11ea-98bd-d87f928f1460.jpg)


Traceback:

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/django/core/handlers/exception.py" in inner
  34.             response = get_response(request)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/django/core/handlers/base.py" in _get_response
  115.                 response = self.process_exception_by_middleware(e, request)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/django/core/handlers/base.py" in _get_response
  113.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/django/views/decorators/csrf.py" in wrapped_view
  54.         return view_func(*args, **kwargs)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/rest_framework/viewsets.py" in view
  114.             return self.dispatch(request, *args, **kwargs)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/rest_framework/views.py" in dispatch
  505.             response = self.handle_exception(exc)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/rest_framework/views.py" in handle_exception
  465.             self.raise_uncaught_exception(exc)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/rest_framework/views.py" in raise_uncaught_exception
  476.         raise exc

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/rest_framework/views.py" in dispatch
  502.             response = handler(request, *args, **kwargs)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/rest_framework/mixins.py" in partial_update
  82.         return self.update(request, *args, **kwargs)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/rest_framework/mixins.py" in update
  68.         self.perform_update(serializer)

File "/var/www/whispersservices_django/whispersservices/views.py" in perform_update
  150.             serializer.save()

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/rest_framework/serializers.py" in save
  207.             self.instance = self.update(self.instance, validated_data)

File "/var/www/whispersservices_django/whispersservices/serializers.py" in update
  4502.                 if validated_data['role'].name in ['SuperAdmin', 'Admin']:

```
Traceback:

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/django/core/handlers/exception.py" in inner
  34.             response = get_response(request)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/django/core/handlers/base.py" in _get_response
  115.                 response = self.process_exception_by_middleware(e, request)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/django/core/handlers/base.py" in _get_response
  113.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/django/views/decorators/csrf.py" in wrapped_view
  54.         return view_func(*args, **kwargs)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/rest_framework/viewsets.py" in view
  114.             return self.dispatch(request, *args, **kwargs)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/rest_framework/views.py" in dispatch
  505.             response = self.handle_exception(exc)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/rest_framework/views.py" in handle_exception
  465.             self.raise_uncaught_exception(exc)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/rest_framework/views.py" in raise_uncaught_exception
  476.         raise exc

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/rest_framework/views.py" in dispatch
  502.             response = handler(request, *args, **kwargs)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/rest_framework/mixins.py" in partial_update
  82.         return self.update(request, *args, **kwargs)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/rest_framework/mixins.py" in update
  68.         self.perform_update(serializer)

File "/var/www/whispersservices_django/whispersservices/views.py" in perform_update
  150.             serializer.save()

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/rest_framework/serializers.py" in save
  207.             self.instance = self.update(self.instance, validated_data)

File "/var/www/whispersservices_django/whispersservices/serializers.py" in update
  4502.                 if validated_data['role'].name in ['SuperAdmin', 'Admin']:
```

json:
```
{
  "new_notification_cue_standard_preferences": [
    {
      "standard_type": 1,
      "new_notification_cue_preference": {
        "create_when_new": false,
        "create_when_modified": true,
        "send_email": false
      }
    },
    {
      "standard_type": 2,
      "new_notification_cue_preference": {
        "create_when_new": false,
        "create_when_modified": false,
        "send_email": false
      }
    },
    {
      "standard_type": 3,
      "new_notification_cue_preference": {
        "create_when_new": false,
        "create_when_modified": false,
        "send_email": false
      }
    },
    {
      "standard_type": 4,
      "new_notification_cue_preference": {
        "create_when_new": false,
        "create_when_modified": false,
        "send_email": false
      }
    }
  ]
}
```

**Comment by aaronstephenson** [comment]
Fixed in c17d7dbd
